components/menu/index.tsx

```diff
@@ -61,7 +61,7 @@ export default class Menu extends React.Component<MenuProps, MenuState> {
   static defaultProps = {
     prefixCls: 'ant-menu',
     className: '',
-    theme: 'light',  // or dark
+    theme: 'light' as 'light' | 'dark' as 'dark',  // or dark
```

Comment:
No double `as`, should be:
```theme: 'light' as 'light' | 'dark'```

Note to head repo maintainer:
I don't think there's a declared (alias) theme type of `'light' | 'dark'` (other than inline) in the current codebase. Might be beneficial to create one if it doesn't exist.